lib/omnibus/version.rb

```diff
@@ -15,5 +15,5 @@
 #
 
 module Omnibus
-  VERSION = '5.0.1'
+  VERSION = '5.0.2'
```

Comment:
I think this should bump to `5.1.0` since we are adding new, backward compatible, features (namely the DSL methods).